src/lib/crypto/signatures.cpp

```diff
@@ -175,6 +194,26 @@ signature_calculate(pgp_signature_t &     sig,
         }
         break;
     }
+#if defined(ENABLE_PQC)
+    case PGP_PKA_DILITHIUM3_ED25519:
+        [[fallthrough]];
+    // TODO: add case PGP_PKA_DILITHIUM5_ED448: [[fallthrough]];
+    case PGP_PKA_DILITHIUM3_P256:
+        [[fallthrough]];
```

Comment:
Recently to eliminate huger number of MSVC warnings the following was added to the main rnp branch:
```
#if (!defined(_MSVC_LANG) || _MSVC_LANG >= 201703L)
        [[fallthrough]];
#endif
```
Could you please that as well?

Reply:
Agree, define would be much better. For the clean changes that would require fixing the already existing codebase, let's leave that for a future separate PR on the main repository, I'll create issue for that.

Reply:
@TJ-91 It's good for now, thanks.